### PR TITLE
add events attributes to table

### PR DESCRIPTION
**Pull request by mmonzo-eb**
I filled the table of events from a mocked file with some fake events. I added some columns to this table

### CODE REVIEW

File: search_events_app/views.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 from django.shortcuts import render
+
```
Comment (walterbucolo-eb) [inline search_events_app/views.py:2]: This line does not go here. The new line is used to separate different groups of imports. (e.g. Django imports and project imports)

File: search_events_app/views.py [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 from django.views.generic.list import ListView
+from .services.dummy_api import DummyApi
```
Comment (walterbucolo-eb) [inline search_events_app/views.py:4]: Here we should put a newline

File: search_events_app/views.py (hunk 1 above)
Comment (mmonzo-eb) [reply search_events_app/views.py]: Already solved

File: search_events_app/views.py (hunk 2 above)
Comment (mmonzo-eb) [reply search_events_app/views.py:4]: Already solved

File: search_events_app/views.py [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
+
 
 # Create your views here.
 class EventListView(ListView):
     template_name = 'event_list.html'
 
     def get_queryset(self):
-        return None
+        d = DummyApi()
+        return d.get()
```
Comment (walterbucolo-eb) [inline search_events_app/views.py:13]: It's possible to return DummyApi().get()?. It does not possible, could you choose a more specific variable name?

Comment (mmonzo-eb) [reply search_events_app/views.py]: Changed to DummyApi().get() :)

**Pull request merged**